src/test/java/de/filefighter/rest/cucumber/ViewFolderContentsSteps.java

```diff
@@ -60,4 +60,42 @@ public void theResponseContainsAnEmptyListForFilesAndFolders() throws JsonProces
 
         assertTrue(rootNode.isEmpty());
     }
+
+    @And("the response not contains the file with fileSystemId {long} and name {string}")
+    public void theResponseNotContainsTheFileWithFileSystemIdAndName(long fsItemId, String name) throws JsonProcessingException {
+
+        ArrayNode rootNode = (ArrayNode) objectMapper.readTree(latestResponse.getBody());
+        if (!rootNode.isContainerNode())
+            throw new AssertionError("Response was not an Array or empty.");
+
+        if
+        (rootNode.isEmpty()) {
+            assertTrue(true);
```

Comment:
what is even that. xd (blame -> @qvalentin )

Reply:
Even more funny is the fact, that neither intellij nor sonar have something against that. xD

Reply:
fixed it.